### ISSUE TITLE
Investigate why Conviction voting came back

### ISSUE THREAD

**Issue by kacperzuk-neti** [liberland/liberland_substrate#224]

**Comment by kacperzuk-neti** [comment]
TL;DR: Looks like Conviction voting was never removed.

How conviction voting works: in Tally.add when recording the vote we call `vote.conviction.votes(balance);` to calculate how many votes given balance is worth given some conviction: https://github.com/liberland/liberland_substrate/blob/main/frame/democracy/src/types.rs#L99.

The `conviction` itself wasn't modified by us ever: https://github.com/liberland/liberland_substrate/commits/main/frame/democracy/src/conviction.rs

The `Tally.add` wasn't modified by us ever: https://github.com/liberland/liberland_substrate/commits/main/frame/democracy/src/types.rs

State of the `lib.rs` at the beginning of November: https://github.com/liberland/liberland_substrate/blob/b7eaeea2a3a55678e16da0ae54a8ad131751aa0f/frame/democracy/src/lib.rs - this is before asset refactor and upstream sync. `tally.add` call is unmodified in `fn try_vote` function, there're no checks or overrides on conviction in the `fn vote` call.

The only thing that was removed was the locking mechanism, so even if we set conviction of vote to Locking6x, our tokens aren't locked (other than Politipool lock), but they do count 6x to the referendum results.

@DorianSternVukotic is it possible that your memory of this working correctly is incorrect? You probably tested it half a year ago, maybe the only thing tested was locking, not vote multiplication.
